packages/ymax-planner/src/engine.ts

```diff
@@ -31,6 +35,8 @@ type CosmosEvent = {
 };
 
 const VSTORAGE_PATH_PREFIX = 'published.ymax0.portfolios';
+const ORCHESTRATION_SUBSCRIPTIONS_PREFIX =
+  'published.orchestration.subscriptions';
```

Comment:
```suggestion
const ORCHESTRATION_SUBSCRIPTIONS_PATH_PREFIX =
  'published.orchestration.subscriptions';
```

Is vstorage path "published.orchestration.subscriptions" limited to YMax? If so, then why the generic name? If not, then where is the relevant filtering?

Also, introducing another path to care about seems to necessitate renaming `VSTORAGE_PATH_PREFIX` to e.g. `PORTFOLIOS_PATH_PREFIX`.

Reply:
Thanks for catching that 🙏. `published.orchestration.subscriptions` was originally specific to YMax. But as of today it was finalized in [#11714 ] to use the path `published.ymax0` instead.

I’ll clean up the naming to reflect this so we don’t have redundant or misleading prefixes.